examples/problems/1100_callback.ipynb

```diff
@@ -0,0 +1,502 @@
+{
```

Comment:
<p>I think therefore we have a different tutorial, right? then just link that example here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/moscot_notebooks/pull/70/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
this is the example for using graphs too